File: src/Entity/Evenement.php
<?php

namespace App\Entity;

use App\Repository\EvenementRepository;
use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
use Symfony\Component\Serializer\Annotation\Groups;
/**
 * @ORM\Entity(repositoryClass=EvenementRepository::class)
 */
class Evenement
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     * @Groups("Reservation")
     */
    private $id;

    /**
     * @ORM\Column(type="float")
     * @Groups("Reservation")
     */
    public $prix_evenement;

    /**
     * @ORM\Column(type="string", length=255)
     * @Groups("Reservation")
     */
    public $decription_evenement;

    /**
     * @ORM\Column(type="string", length=255)
     * @Groups("Reservation")
     */
    public $titre_evenement;

    /**
     * @ORM\Column(type="integer")
     * @Groups("Reservation")
     */
    public $nbre_places;

    /**
     * @ORM\Column(type="date")
     * @Groups("Reservation")
     */
    public $date_debut;

    /**
     * @ORM\Column(type="date")
     * @Groups("Reservation")
     */
    public $date_fin;



    /**
     * @ORM\OneToMany(targetEntity=Reservation::class, mappedBy="even", cascade={"all"}, orphanRemoval=true)
     */
    private $idRes;


    public function __construct()
    {
        $this->detailEvens = new ArrayCollection();
        $this->reservations = new ArrayCollection();
        $this->idRes = new ArrayCollection();
    }

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getPrixEven(): ?float
    {
        return $this->prix_evenement;
    }

    public function setPrixEven(float $prix_even): self
    {
        $this->prix_evenement = $prix_even;

        return $this;
    }

    public function getDescEven(): ?string
    {
        return $this->decription_evenement;
    }

    public function setDescEven(string $desc_even): self
    {
        $this->decription_evenement = $desc_even;

        return $this;
    }

    public function getTitreEven(): ?string
    {
        return $this->titre_evenement;
    }

    public function setTitreEven(string $titre_even): self
    {
        $this->titre_evenementen = $titre_even;

        return $this;
    }

    public function getNbrePlace(): ?int
    {
        return $this->nbre_places;
    }

    public function setNbrePlace(int $nbre_place): self
    {
        $this->nbre_places = $nbre_place;

        return $this;
    }

    public function getDateDebut(): ?\DateTimeInterface
    {
        return $this->date_debut;
    }

    public function setDateDebut(\DateTimeInterface $date_debut): self
    {
        $this->date_debut = $date_debut;

        return $this;
    }

    public function getDateFin(): ?\DateTimeInterface
    {
        return $this->date_fin;
    }

    public function setDateFin(\DateTimeInterface $date_fin): self
    {
        $this->date_fin = $date_fin;

        return $this;
    }


    /**
     * @return Collection|Reservation[]
     */
    public function getIdRes(): Collection
    {
        return $this->idRes;
    }

    public function addIdRe(Reservation $idRe): self
    {
        if (!$this->idRes->contains($idRe)) {
            $this->idRes[] = $idRe;
            $idRe->setEven($this);
        }

        return $this;
    }

    public function removeIdRe(Reservation $idRe): self
    {
        if ($this->idRes->removeElement($idRe)) {
            // set the owning side to null (unless already changed)
            if ($idRe->getEven() === $this) {
                $idRe->setEven(null);
            }
        }

        return $this;
    }



    public function __toString()
    {
        return $this-> decription_evenement;
    }

}

File: TuniDiscoveryMobile/src/com/mycompany/myapp/entities/Evenement.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

import java.util.Date;

/**
 *
 * @author Lenovo
 */
public class Evenement {
    private int id;
    private float prix_even;
    private String desc_even;
    private String titre_even;
    private int nbre_place;
    private String date_debut;
    private String date_fin;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public float getPrix_even() {
        return prix_even;
    }

    public void setPrix_even(float prix_even) {
        this.prix_even = prix_even;
    }

      public String getputostring ()
    {
        return String.valueOf(prix_even);
    }
    public String getDesc_even() {
        return desc_even;
    }

    public void setDesc_even(String desc_even) {
        this.desc_even = desc_even;
    }

    public String getTitre_even() {
        return titre_even;
    }

    public void setTitre_even(String titre_even) {
        this.titre_even = titre_even;
    }

    public int getNbre_place() {
        return nbre_place;
    }

    public void setNbre_place(int nbre_place) {
        this.nbre_place = nbre_place;
    }

    public String getDate_debut() {
        return date_debut;
    }

    public void setDate_debut(String date_debut) {
        this.date_debut = date_debut;
    }

    public String getDate_fin() {
        return date_fin;
    }

    public void setDate_fin(String date_fin) {
        this.date_fin = date_fin;
    }
    
      public String nbplacetostring ()
    {
        return String.valueOf(nbre_place);
    }

    @Override
    public String toString() {
        return "Evenement{" + "id=" + id + ", prix_even=" + prix_even + ", desc_even=" + desc_even + ", titre_even=" + titre_even + ", nbre_place=" + nbre_place + ", date_debut=" + date_debut + ", date_fin=" + date_fin + '}';
    }

    public Evenement() {
    }

    public Evenement(int id, float prix_even, String desc_even, String titre_even, int nbre_place, String date_debut, String date_fin) {
        this.id = id;
        this.prix_even = prix_even;
        this.desc_even = desc_even;
        this.titre_even = titre_even;
        this.nbre_place = nbre_place;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
    }

                            
                            
}
